sim/core/consumes.go

```diff
@@ -498,9 +501,10 @@ func DragonBreathChiliAura(unit *Unit) *Aura {
 }
 
 // Creates a spell object for the common explosive case.
+// TODO: create 10s delay on Goblin Landmine cast to damage
```

Comment:
A 10s activation can be done by making a Dot for the spell with a single tick of 10s. Refer to other Dot spells as needed.